content/en/docs/guides/development/localization/localization-guide.md

```diff
@@ -196,7 +207,7 @@ Here's an example for webOS application.
 ```
 {{< /code >}}
 
-##### notes
+##### note
```

Comment:
If you use note in the document, please use **{{< note >}} note content {{< /note >}}** style.